package.json

```diff
@@ -24,9 +24,10 @@
     "@changesets/cli": "^2.26.0"
   },
   "volta": {
-    "node": "16.15.1"
+    "node": "16.15.1",
+    "pnpm": "7.3.0"
   },
-  "packageManager": "pnpm@7.1.9",
+  "packageManager": "pnpm@7.3.0",
```

Comment:
It's 7.3.0 over here: https://github.com/CrowdStrike/ember-toucan-styles/blob/c8377460d2737639e063d9b99b6db3c8d32e1ffd/ember-toucan-styles/package.json#L115